src/features/auth/login/LoginForm.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState, type FC } from 'react';
+import { Form, Formik } from 'formik';
+import { observer } from 'mobx-react';
+import * as Yup from 'yup';
+import { AuthInput } from '@entities/auth/AuthInput';
+import { Button } from '@shared/ui/Button';
+import { Icon } from '@shared/icons/Icon';
+import { useAuth } from '../model/useAuth';
+import type { LoginOptions } from './interfaces/login-options.interface';
+
+export const LoginForm: FC = observer(() => {
+  const [isVisiblePassword, setIsVisiblePassword] = useState(false);
+  const { handleSubmitLogin } = useAuth();
+
+  const LoginSchema = Yup.object().shape({
+    email: Yup.string().email('Invalid email').required('Required'),
+    password: Yup.string()
+      .min(8, 'Password must be at least 8 characters')
```

Comment:
Shouldn't use magic number
```suggestion
      const MIN_CHARS = 8;
      .min(MIN_CHARS, 'Password must be at least 8 characters')
```